README.md

```diff
@@ -31,7 +31,9 @@ sourced under [Apache version 2.0](LICENSE).
 To read more about Firecracker, check out
 [firecracker-microvm.io](https://firecracker-microvm.github.io).
 
-## Getting Started
+## Getting Started Building Firecracker
+
+Note: If you just want to run Firecracker see the Releases section below.
 
 You can build Firecracker on any system that has Docker running (we use a
 development container). The simple steps to get & build Firecracker are:
```

Comment:
This can also be changed to:
```
You can build Firecracker on any system that has Docker running (we use a
development container) as follows:
```

Reply:
Updated.